packages/composite-checkout-wpcom/src/components/wp-order-review-line-items.js

```diff
@@ -10,7 +10,13 @@ import { useTranslate } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+<<<<<<< HEAD:packages/composite-checkout-wpcom/src/components/wp-order-review-line-items.js
```

Comment:
Just noticed this... cleaning it up.

Reply:
No worries. I'm surprised the precommit linter let that through. 🤔